src/view/DisplayQuizzesView.java

```diff
@@ -0,0 +1,150 @@
+import interface_adapter.quiz.display.DisplayQuizzesState;
+import interface_adapter.quiz.display.DisplayQuizzesViewModel;
+import interface_adapter.quiz.take_quiz.TakeQuizController;
+import interface_adapter.quiz.take_quiz.TakeQuizPresenter;
+import interface_adapter.quiz.take_quiz.TakeQuizState;
+import interface_adapter.quiz.take_quiz.TakeQuizViewModel;
+import use_case.quiz.QuizDataAccessInterface;
+import use_case.quiz.take_quiz.TakeQuizInputBoundary;
+import use_case.quiz.take_quiz.TakeQuizInteractor;
+import use_case.quiz.take_quiz.TakeQuizOutputBoundary;
+
+import javax.swing.*;
+import javax.swing.event.ListSelectionEvent;
+import javax.swing.event.ListSelectionListener;
+import javax.swing.table.DefaultTableModel;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+
+public class DisplayQuizzesView extends JPanel implements PropertyChangeListener {
+    public final String viewName = "quiz";
+
+    /** The column names for the quiz table. */
+    String[] columnNames = {"Quiz Title", "No. Questions", "Time Created" };
+
+    /** The data to be displayed in the quiz table. */
+    private String[][] quizData;
+    private final DisplayQuizzesController displayQuizzesController;
+    private final DisplayQuizzesViewModel displayQuizzesViewModel;
+
+    JTable table;
+    final JButton start;
+    final JButton edit;
+    final JButton back;
+
+    private ListSelectionModel selectionModel;
+    private String selectedTitle;
+
+    public DisplayQuizzesView(DisplayQuizzesViewModel displayQuizzesViewModel,
+                              DisplayQuizzesController controller,
+                              TakeQuizController takeQuizController,
+                              TakeQuizViewModel takeQuizViewModel) {
+        this.displayQuizzesViewModel = displayQuizzesViewModel;
+        this.displayQuizzesController = controller;
+
+
+        displayQuizzesViewModel.addPropertyChangeListener(this);
+
+        Box buttons = Box.createVerticalBox();
+        start = new JButton(DisplayQuizzesViewModel.START_LABEL);
+        back = new JButton(DisplayQuizzesViewModel.BACK_LABEL);
+        edit = new JButton(DisplayQuizzesViewModel.EDIT_LABEL);
+        buttons.add(start);
+        buttons.add(Box.createVerticalStrut(10));
+        buttons.add(back);
+        buttons.add(Box.createVerticalStrut(10));
+        buttons.add(edit);
+        buttons.add(Box.createVerticalStrut(10));
+
+
+        start.addActionListener(
+                new ActionListener() {
+                    @Override
+                    public void actionPerformed(ActionEvent e) {
+                        if (e.getSource().equals(start)) {
+
+                            // TODO: implement start quiz button
+                            // get the quiz id OR title, execute on that
+                            // change screen to the questionview
```

Comment:
I don't quite get the "getting the ID" part. Does this work regardless of getting the ID or the title?